CHANGES.md

```diff
@@ -7,6 +7,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.3.0] - 2021-12-10
+
+### Added
+
+- A precompile which exposes NEAR's random number generator was added by [@mfornet] as requested by 
+[@birchmd]. ([#368] [#297])
+- London hard fork support was added by [@birchmd]. ([#244])
+
+### Changes
+
+- The gas limit for `deposit` and `ft_on_transfer` were changed as they were not attaching enough
+gas, as changed by [@mrLSD]. ([#389])
+
+### Fixes
+
+- There was an issue with the original storage not actually being stored. Unfortunately, previous 
+transactions can't be updated with this change. This has been fixed by [@birchmd]. ([#390])
```

Comment:
This is a pretty poor description that will not be comprehensible to most anybody who reads this changelog. What is "the original storage"? Cool, old transactions can't be updated--what is the actual impact?